src/main/java/com/app/folioman/mfschemes/config/MfApiProperties.java

```diff
@@ -0,0 +1,25 @@
+package com.app.folioman.mfschemes.config;
+
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.Pattern;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.validation.annotation.Validated;
+
+@Validated
+@ConfigurationProperties(prefix = "app.nav.mfapi")
+public class MfApiProperties {
+
+    @NotBlank(message = "Data URL must not be blank")
+    @Pattern(
+            regexp = "^https?://[-a-zA-Z0-9+&@#/%?=~_|!:,.;]*[-a-zA-Z0-9+&@#/%=~_|]",
```

Comment:
@coderabbitai, please change the regex that works with a valid url https://api.mfapi.in/mf/{schemeCode}